File: official/docs/java/reports/RetrieveReport.java
package reports;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.Report;

public class RetrieveReport {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        Report report = Report.retrieve("<REPORT_ID>");

        System.out.println(report);
    }
}

File: official/docs/go/reports/retrieve_report.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	report, _ := client.GetReport("<REPORT_TYPE>", "<REPORT_ID>")

	fmt.Println(report)
}
